Decent.Minecraft.Client/RailType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Decent.Minecraft.Client
+{
+    /// <summary>
+    /// The type of rail.
+    /// </summary>
+    public enum RailType : byte
```

Comment:
Ah, I was thinking of the other enum I added. I hit a bunch of issues while trying to make things derived types because Powered Rails are not just Rails with Power. I'll think about it some more.